cli_tools/common/utils/daisyutils/workflow_hook_fallback_pd.go

```diff
@@ -0,0 +1,62 @@
+//  Copyright 2021 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package daisyutils
+
+import (
+	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/logging"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+)
+
+// FallbackToPDStandard detects if a workflow fails due to insufficient SSD
+// quota. If so, it re-runs the workflow and modifies the disks to use PD Standard.
+type FallbackToPDStandard struct {
+	logger   logging.Logger
+	fallback bool
```

Comment:
Maybe `shouldFallback` or `doFallback`? Just `fallback` sounds like it's a property of the Fallback object, which sounds strange, while above suggestions are obviously instructions to perform fallback.

Reply:
SGTM. Changed to `shouldFallback`.